src/compiler/program.ts

```diff
@@ -586,6 +584,13 @@ namespace ts {
         // used to track cases when two file names differ only in casing
         const filesByNameIgnoreCase = host.useCaseSensitiveFileNames() ? createMap<SourceFile>() : undefined;
 
+        const referencedProjectOutFiles: Path[] = [];
+        const projectReferenceRedirects = createProjectReferenceRedirects(options);
+        checkProjectReferenceGraph();

```

Comment:
can we do this in `verifyCompilerOptions` instead.

Reply:
✔️